runatlantis.io/docs/repo-level-atlantis-yaml.md

```diff
@@ -56,6 +56,7 @@ projects:
   workspace: default
   terraform_version: v0.11.0
   delete_source_branch_on_merge: true
+  disable_repo_locking: true
```

Comment:
Instead of a disable key set to false, id much prefer a simpler repo key like `repo_locking: true`

We only use the enable and disable prefixes in the server flags and I'd much prefer the former with an adequate default than the latter.

Thoughts on naming of this key @runatlantis/maintainers

Reply:
Renamed https://github.com/runatlantis/atlantis/pull/2700/commits/9d52bbc886ee5d66f8e8734b3f7fbb4049d34202

Dand it works fine on local with https://github.com/krrrr38/atlantis-debug/pull/1/commits/4080b2453dc64d59c43699ec9c32c0bc5cbd4c05